### DISCUSSION TITLE
When compiling to docx, how do I specify what word style is applied?

### DISCUSSION THREAD

**Discussion by rcyost** [quarto-dev/quarto-cli#10241] [Q&A]
### Description

````{qmd}
- This text gets styled with "compact" word style.

| col1 |
|-----|
| text |   
````

![image](https://github.com/quarto-dev/quarto-cli/assets/32645001/3345dba5-602a-4e79-bea9-e1829a786322)
![image](https://github.com/quarto-dev/quarto-cli/assets/32645001/43f9416f-7de5-4aaf-9216-36bece120676)

The text in the table also gets the compact styling. 
Thus I cannot figure out how to apply a custom table formatting that does not get applied to the text written in the .qmd file when it is compiled to docx. 
Is there a way to tell quarto which style from the custom reference file to use for the cell or text? 

Thank you. 

````

**Comment by mcanouil** [comment]
See:
- https://quarto.org/docs/output-formats/ms-word-templates.html
